4_Bayes_Rule/Bayes_Rule_All.ipynb

```diff
@@ -0,0 +1,4120 @@
+{
```

Comment:
<p>I love the link to regularization and I think the example works well, but it could still be a bit confusing if people know the example. Because usually you would just pick a degree 4 polynomial via model selection, while in your example you implicitly state that we'd have all polynomials and assign priors that force higher ones to be really small (which would definitely work).</p><p> </p><p>A better example could be a regression with many regressors and we want most of them to be small. Unfortunately I don't know how to associate that with a plot that has the some power as yours. So I don't really have a great alternative.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>yea this is a good point, maybe I should bring up the AB test example and say, we know conversion wont be under 0 or over 100, or something else. Ill think on it and update</p>

Reply:
<p>Yes, conversions from A/B would be good. You can also use the IQ simulation we have in the workshops to illustrate. Sounds interesting?</p>

Reply:
Can you share IQ simulation? Thats sounds like a good one